common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -105,10 +106,14 @@ class CapaFields(object):
     max_attempts = Integer(
         display_name=_("Maximum Attempts"),
         help=_("Defines the number of times a student can try to answer this problem. "
-               "If the value is not set, infinite attempts are allowed."),
+               "If the value is not set, infinite attempts are allowed. "
+               "NOTE: If a problem is timed, we only allow a single attempt, and ignore "
+               "the value in this field."),
         values={"min": 0}, scope=Scope.settings
     )
-
+    minutes_before_warning = Integer(
+        help=_("Number of minutes at which the student will be issued a timed warning. Works only when minutes_allowed is non-zero"),
+        default=1, scope=Scope.settings)
```

Comment:
Please each kwarg on its own line, add a trailing comma after the final param, and drop the closing paren to its own line.